src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IInvocationOperation.cs

```diff
@@ -1312,5 +1313,81 @@ void M(bool b, object o1, object o2, object o3, object o4, object o5)
 ";
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedFlowGraph, expectedDiagnostics);
         }
+
+        [Theory, WorkItem(51715, "https://github.com/dotnet/roslyn/issues/51715")]
+        [InlineData("static")]
+        [InlineData("")]
+        public void Invocation_LocalFunction_01(string modifier)
```

Comment:
>Invocation_LocalFunction_01 [](http://example.com/codeflow?start=20&length=27)

It looks like all the tests are testing scenarios within an implicit static method. Consider testing inside a regular instance method and, perhaps, a regular static method as well.